tools/test/test-setup.sh

```diff
@@ -24,7 +24,8 @@ if [[ "$1" = "--no_echo" ]]; then
   no_echo="true"
   shift
 else
-  echo 'exec ${PAGER:-/usr/bin/less} "$0" || exit 1'
+  echo 'exec ${PAGER:-/usr/bin/less1} "$0" || exit 1'
```

Comment:
Is `less1` a typo? I don't have that installed.

Reply:
Otherwise LGTM.

Reply:
Grr, must have typoed in there. Thanks for the catch.